core/db/db_test.go

```diff
@@ -31,246 +31,104 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestGetDBPathEmptyPath(t *testing.T) {
-	originalSetting := viper.GetString("peer.fileSystemPath")
-	viper.Set("peer.fileSystemPath", "")
+func TestCreateDB(t *testing.T) {
+	openchainDB := Create()
+	openchainDB.Open()
+	defer deleteTestDBPath()
+	defer openchainDB.Close()
+}
+
+func TestOpenDB_DirDoesNotExist(t *testing.T) {
```

Comment:
irrespective of the return type, this test actually creates rocksDB files in an empty directory so, the previous name was more meaningful.